gatsby-plugin-s3-e2e-tests-infrastructure/test-policy.json

```diff
@@ -20,7 +21,9 @@
             "Effect": "Allow",
             "Action": [
                 "s3:GetObject",
-                "s3:DeleteObject"
+                "s3:DeleteObject",
+                "s3:PutObject",
```

Comment:
can this be removed in favor of line 26, `s3:PutObject*` ?

Reply:
Done :D